src/bitmessageqt/__init__.py

```diff
@@ -2393,6 +2393,8 @@ def click_actionSettings(self):
                 QMessageBox.about(self, _translate("MainWindow", "Number needed"), _translate(
                     "MainWindow", "Your maximum download and upload rate must be numbers. Ignoring what you typed."))
 
+            shared.config.set('bitmessagesettings', 'maxoutboundconnections', str(
```

Comment:
Refer to the immediately preceding options to see how to ensure the user entered a number.